mmv1/products/compute/terraform.yaml

```diff
@@ -2845,6 +2845,44 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_value: :TLS_1_0
       warnings: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
+  # Find that Region SSL Policy needs the same custom code as SSL Policy
+  RegionSslPolicy: !ruby/object:Overrides::Terraform::ResourceOverride
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      constants:  'templates/terraform/constants/region_ssl_policy.erb'
+      resource_definition: 'templates/terraform/resource_definition/region_ssl_policy.erb'
+      update_encoder: 'templates/terraform/update_encoder/region_ssl_policy.erb'
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      customFeatures: !ruby/object:Overrides::Terraform::PropertyOverride
+        is_set: true
+        # TODO(https://github.com/GoogleCloudPlatform/magic-modules/issues/187) replace references automatically
+        # Overriding the whole description because original has references to
+        # other fields which are in camelCase in api.yaml.
+        description: |
+          Profile specifies the set of SSL features that can be used by the
+          load balancer when negotiating SSL with clients. This can be one of
+          `COMPATIBLE`, `MODERN`, `RESTRICTED`, or `CUSTOM`. If using `CUSTOM`,
+          the set of SSL features to enable must be specified in the
+          `customFeatures` field.
```

Comment:
```suggestion
          `custom_features` field.
```
is this the camel case you were trying to overwrite?

Reply:
I borrowed this from the `ssl_policy` (global) section. Thinking it can be cleaned up